packages/connect-thegraph/src/core/GraphQLWrapper.ts

```diff
@@ -61,7 +34,10 @@ export default class GraphQLWrapper {
     const request = createRequest(query, args)
 
     return pipe(
-      this.#client.executeSubscription(request),
+      this.#client.executeQuery(request, {
+        pollInterval: POLL_INTERVAL,
```

Comment:
I am wondering if we should have an option for this on `connect()`?

For example:

```
const org = await connect('myorg', 'thegraph', { pollInterval: 20000 })
```

The day we move back to GraphQL subscriptions, we could make it `-1` by default, so that any change on this value could keep using polling (it might be useful to support both).

Reply:
Yeah, having this as a configurable option sounds good! I was thinking that we should also could make this standard for the connectors, so we can have granularity over which connectors we may want to set a polling interval.

Reply:
Yes, I think it makes sense to have it as an option on `connect()`. I also agree with Enrique suggestion of having it on connectors. As mentioned above maybe adding it as a new argument on the GraphQLWrapper constructor?

Reply:
Done in https://github.com/aragon/connect/pull/203/commits/171161d904116902fbacc801d7eaeb7566d20888